packages/mrml-python/src/lib.rs

```diff
@@ -42,8 +42,8 @@ pub struct HttpIncludeLoaderOptions {
     list: HashSet<String>,
 }
 
-// #[pyclass]
-#[derive(FromPyObject, Clone, Debug)]
+#[pyclass]
+#[derive(Clone, Debug)]
```

Comment:
Is there a reason why `#[pyclass]` was not used? It seems to work well in the new pyo3 version.